arrow/src/test/java/org/apache/calcite/adapter/arrow/ArrowAdapterTest.java

```diff
@@ -325,23 +325,29 @@ static void initializeArrowState(@TempDir Path sharedTempDir) throws IOException
         .explainContains(plan);
   }
 
+  /** <a href="https://issues.apache.org/jira/browse/CALCITE-6296">[CALCITE-6296]</a>. */
   @Test void testArrowProjectFieldsWithIsNullFilter() {
     String sql = "select \"intField\", \"stringField\"\n"
         + "from arrowdata\n"
         + "where \"intField\" is null";
-    String plan;
-    if (Bug.CALCITE_6296_FIXED) {
-      plan = "ArrowToEnumerableConverter\n"
+    String plan = "ArrowToEnumerableConverter\n"
           + "  ArrowProject(intField=[$0], stringField=[$1])\n"
-          + "    ArrowFilter(condition=[IS NOT NULL($0)])\n"
-          + "      ArrowTableScan(table=[[ARROW, ARROWDATA]], fields=[[0, 1, 2, 3]])\n\n";
-    } else {
-      plan = "PLAN=EnumerableCalc(expr#0..1=[{inputs}],"
-          + " expr#2=[IS NULL($t0)], proj#0..1=[{exprs}], $condition=[$t2])\n"
-          + "  ArrowToEnumerableConverter\n"
-          + "    ArrowProject(intField=[$0], stringField=[$1])\n"
+          + "    ArrowFilter(condition=[IS NULL($0)])\n"
           + "      ArrowTableScan(table=[[ARROW, ARROWDATA]], fields=[[0, 1, 2, 3]])\n\n";
-    }
+
+    CalciteAssert.that()
+        .with(arrow)
+        .query(sql)
+        .returnsCount(0)
+        .explainContains(plan);
+  }
+
+  @Test void testConjunctiveIsNullFilters() {
```

Comment:
Let's add the jira link here too, for uniformity

Reply:
[Add test case comment to testConjunctiveIsNullFilters](https://github.com/apache/calcite/pull/3858/commits/634467a9e6bf98db8d4cf07f9fab19edf86eec0c)